warcry_proj/README.txt

```diff
@@ -0,0 +1,22 @@
+warcry_proj README
+==================
+
+Getting Started
+---------------
+
+- Login to Vagrant
+* vagrant up
+* vagrant ssh
+
+- Ensure the bootstrap script is run to configure and install all required packages
+* sudo script/bootstrap vagrant
```

Comment:
cool, ill move all the project dependant pip installs to a file called project.yml